h/indexer.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from h.celery import celery
+
+from h.api import storage
+from h.api.search.index import index
+from h.api.search.index import delete
+
+__all__ = (
+    'add_annotation',
+    'delete_annotation',
+)
+
+
+@celery.task
+def add_annotation(id_):
+    annotation = storage.fetch_annotation(celery.request, id_)
```

Comment:
#3250 is now merged. I've rebased this branch and fixed a small issue due to the `AnnotationEvent` not containing the `h.api.model.Annotation` object anymore but the serialized version of an annotation. This is now ready for review again.